assets/texts/pt-br/Changelog/1.2.html

```diff
@@ -0,0 +1,9 @@
+<ul>
```

Comment:
You don't need to include files you did not translate, so feel free to delete these untranslated files here.

Reply:
Thanks for the tip! I will leave as a reminder to myself to translate it later if that's OK. This is indeed an opportunity for me to work together with other people in GitHub for the 1st time, so I am learning things. Please feel free to point out best practices, I have read some stuff but I know I am probably gonna make some mistakes further on.